src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -211,4 +213,24 @@ struct MassFlux : db::SimpleTag {
   static std::string name() { return Frame::prefix<Fr>() + "MassFlux"; }
 };
 }  // namespace Tags
+
+/// %Tags for options of hydrodynamic systems.
+namespace OptionTags {
+
+/// The equation of state of the fluid.
+template <size_t ThermoDim = 0, bool IsRelativistic = true>
+struct EquationOfState {
+  using type = std::unique_ptr<
+      EquationsOfState::EquationOfState<ThermoDim, IsRelativistic>>;
+  static constexpr Options::String help = {"The equation of state to use"};
+};
+
+// TODO : make the equation of state base class generic
```

Comment:
I think our CI flags `TODO` comments. I'd say you can just remove this chunk of code for now and implement it in a followup PR. No need to hold up the great work here for another feature :D